_posts/2018-03-08-roadmap.md

```diff
@@ -0,0 +1,221 @@
+---
+layout: post
+title: "Rust's 2018 roadmap"
```

Comment:
ultimate nit of all nits: This could have a nice unicode quotation mark, the same one that’s in the first heading two-three lines below :)

Reply:
This is actually a yaml document, so the should be ascii quotation marks.

Reply:
The apostrophe is what I had in mind. It should not be a part of YAML syntax.